blade/procData/views.py

```diff
@@ -18,3 +20,32 @@ def breachStructure_detail(request, email):
 	return render(request,
 		'breachStructure/detail.html',
 		{'breachStructures':breachStruct})
+
+#def query- username
+def upload_csv(request):
+	if request.method == "POST":
+		form = uploadCSV(request.POST, request.FILES)
+		if form.is_valid():
+			csv_file = request.FILES['csv_file']
+
+			reader = DictReader(csv_file.read().decode('utf-8').splitlines())
+			for row in reader:
+				print(row)
+				#dbQueriedUsers.objects.create(user=row[0])
```

Comment:
yes this is how you would make a new row in the table

Reply:
you can also use get_or_create to prevent duplicate entries, maybe check this before?

you can also use bulk_create to do the insert of various ones in one batch commit, would be faster, but with this small amount of data the performance gain is negligible